app/models/support_day.rb

```diff
@@ -1,8 +1,23 @@
 class SupportDay < ApplicationRecord
   SUPPORT_TYPES = %w[dev ops ooh1 ooh2].freeze
+  MONTHS_IN_THE_PAST = 2
+  TOTAL_MONTHS = 12
 
   belongs_to :team_member
 
   validates_presence_of :date
   validates_presence_of :support_type
+
+  default_scope -> { order(date: :asc) }
+
+  scope :current, -> { where("? <= date AND date <= ?", interval_starting_wednesday.first, interval_starting_wednesday.last) }
+
+  def self.interval_starting_wednesday(date = Date.today)
+    interval_start = date << MONTHS_IN_THE_PAST
+    until interval_start.wednesday?
+      interval_start += 1
+    end
+
+    [interval_start, interval_start + TOTAL_MONTHS.months]
+  end
```

Comment:
I don't understand what this is supposed to be doing? Why are months involved?

Reply:
We need to show at least the month before the current week, for bob to check who was on OoH and thus needs extra pay, and we need to show some months in the future, so bob can check how well they're covered for support.

dxw weeks start on Wednesdays, so starting on a Wednesday in the past is going to make for the neatest tabular display-by-week.

Would it help if I added this to the commit message?

Reply:
Probably. I guess it's also very different to the way the other view works is why I was confused.